README.md

```diff
@@ -64,12 +64,30 @@ the response error message.
 ### Download
 
 Fosdick, once shipments have been shipped, deposit a text file back into the S3 bucket.
-There is a simple, rake runable factory that wraps the logic to do the following:
 
-1. Download the content from these shipment manifests.
-1. Parse this info into arrays of strings
-1. Utilize [specific parts of the array][1] to update relevant shipments
-1. Profit?
+To download and process those files you need to do something like this:
+
+```ruby
+  # download all the keys from S3 that contain 'ship' and '.txt' in the key
+  # we return only the keys so it's easy to feed each key into a worker
+  keys = ShipFosdick::Downloader.download
+  MySampleWorker.perform_async(key)
+
+
+  # The worker could be something like this:
+  class MySampleWorker
+    include Sidekiq::Worker
```

Comment:
Should we rely on the host apps to have access to ActiveJob? Obviously not now due to the time crunch but wonder if this is worthwhile in the future as a refactor.